### PR TITLE
Register @croz/plugin-ibm-apic-backend plugin

### PR DESCRIPTION
**Pull request by djajcevic**
## Hey, I just made a Pull Request!

Registering a `@croz/plugin-ibm-apic-backend` plugin.

### CODE REVIEW

File: microsite/data/plugins/ibm-apic-backend.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+category: API Management
+description: Bring IBM APIC to Backstage.
+documentation: https://croz.net/news/backstage-plugin-ibm-apic?utm_source=backstage.io&utm_medium=marketplace&utm_campaign=backstage-ibm-apic-backend
+iconUrl: https://croz.net/wp-content/uploads/2024/02/croz_large-o.png
+npmPackageName: '@croz/plugin-ibm-apic-backend'
+tags:
+  - idp
+  - developer
+  - portal
+  - internal
```
Comment (awanlin) [inline microsite/data/plugins/ibm-apic-backend.yaml:14]: It's on a dedicated Backstage directory so it's all of these aren't needed - idp, developer, portal, internal.

File: microsite/data/plugins/ibm-apic-backend.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: IBM APIC Backend
+author: croz.net
+authorUrl: https://croz.net/?utm_source=backstage.io&utm_medium=marketplace&utm_campaign=backstage-ibm-apic-backend
+category: API Management
+description: Bring IBM APIC to Backstage.
+documentation: https://croz.net/news/backstage-plugin-ibm-apic?utm_source=backstage.io&utm_medium=marketplace&utm_campaign=backstage-ibm-apic-backend
```
Comment (awanlin) [inline microsite/data/plugins/ibm-apic-backend.yaml:7]: Ideally this should go to the repo `README` or a dedicated documentation page and not a blog post.

File: microsite/data/plugins/ibm-apic-backend.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: IBM APIC Backend
```
Comment (awanlin) [inline microsite/data/plugins/ibm-apic-backend.yaml:2]:
```suggestion
title: IBM APIC EntityProvider
```

Would that make more sense? Seems more clear to me as an outsider to IBM APIC

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for the contribution @djajcevic, left a few comments.

I do have a few general additional items of note:

- Where is the code for this? You didn't use it in the documentation link and it's not linked in the NPM package. Having this does provide the ability for potential Adopters to see the code before they use it and a light trust from a security perspective.
- It's also not clear what you get from this plugin, the only image you share is what seems like a very custom UI and the plugin does not have a frontend.

File: microsite/data/plugins/ibm-apic-backend.yaml (hunk 1 above)
Comment (djajcevic) [reply microsite/data/plugins/ibm-apic-backend.yaml:14]: I've just copied the example from the docs. Will remove those.

File: microsite/data/plugins/ibm-apic-backend.yaml (hunk 3 above)
Comment (djajcevic) [reply microsite/data/plugins/ibm-apic-backend.yaml:2]: I agree with you. Will update the name.

File: microsite/data/plugins/ibm-apic-backend.yaml (hunk 1 above)
Comment (awanlin) [reply microsite/data/plugins/ibm-apic-backend.yaml:14]: OK, can you point me to the docs you used? This is where I generally send people but I don't see it there - https://backstage.io/docs/plugins/add-to-directory 🤔

Comment (djajcevic) [reply microsite/data/plugins/ibm-apic-backend.yaml:14]: Oh :) My bad. I saw it in the existing plugins eq. https://github.com/backstage/backstage/blob/master/microsite/data/plugins/argo-cd.yaml.
I'll remove those.

File: microsite/data/plugins/ibm-apic-backend.yaml (hunk 2 above)
Comment (djajcevic) [reply microsite/data/plugins/ibm-apic-backend.yaml:7]: I'm working we my team, will update the discussion when ready.

**Comment by djajcevic** [comment]
@awanlin Thank you for your awesome and quick responses. I've updated the PR information.

File: microsite/data/plugins/ibm-apic-backend.yaml (hunk 1 above)
Comment (awanlin) [reply microsite/data/plugins/ibm-apic-backend.yaml]: Ok, that's fair then. We should probably remove them at some point. The tags actually aren't used for anything right now just the category.

**Review by awanlin — APPROVED** [review]
Thanks for submitting this plugin @djajcevic, looks good 👍

**Comment by djajcevic** [comment]
Heya all!

I would want to reopen this PR.

What to do next?

**Review by freben — APPROVED** [review]
Sorry for the delay!

**Pull request merged**